orm/hibernate-orm-4/src/test/java/org/hibernate/bugs/ORMStandaloneJpaUnitTestCase.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2014 JBoss Inc
```

Comment:
We usually don't add years anymore (or full headers, for that matter). It's just too tedious to maintain and it's not needed. E.g. that's what we have in Validator (which is also ASF-2-licensed):

/*
 * Hibernate Validator, declare and validate application constraints
 *
 * License: Apache License, Version 2.0
 * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
 */

I'd do it similarly for the test case template project.